src/main/java/seedu/address/model/person/Person.java

```diff
@@ -135,6 +139,40 @@ public static void syncMeetingTimes(Person toSync) {
         toSync.meetingTimes.removeIf(predicate);
     }
 
+    /**
+     * Returns the earliest meeting time from the person's meeting times.
+     * @return The earliest Meeting Time object
+     */
+    public MeetingTime getEarliestMeeting() {
+        Set<MeetingTime> copyMeetingTimes = new HashSet<>();
+        copyMeetingTimes.addAll(meetingTimes);
+        MeetingTime earliestMeeting = removeFirst(copyMeetingTimes);
+        for (int i = 0; copyMeetingTimes.size() > 0; i++) {
+            MeetingTime temp = removeFirst(copyMeetingTimes);
+            if (temp.getDate().isBefore(earliestMeeting.getDate())) {
+                earliestMeeting = temp;
+            }
+        }
+        return earliestMeeting;
+    }
```

Comment:
Much better to return the MeetingTime instead of a Set<MeetingTime>
Thanks shaune <3